lib/pool.js

```diff
@@ -16,6 +16,14 @@ function parseUrl(url) {
   throw new Error('url is invalid: ' + url)
 }
 
+function unbuffer (key, value) {
```

Comment:
The one in `lib/log` iterates over an object's keys and unbuffers, this one unbuffers a single value. It did occur to me that `lib/log` should be refactored to use the function from `lib/crypto/butil`, but that seemed unrelated to this change. Whether they/it should be refactored to call something like this per-property is a good question though.

*Thinks for a moment.*

Yep, good shout, I'll make the change.